src/css/base/base.css

```diff
@@ -111,3 +111,80 @@ main:has(.back-link) {
     height: 20px;
   }
 }
+
+/* #region Logo */
+/* Logo, reusable part */
+.logo {
+  color: var(--green-color);
+  font-weight: 700;
+  font-size: 20px;
+}
+/* #endregion */
+
+/* #region Nav links */
+/* Navigation links, reusable part */
+.nav_link {
+  /* to power pseudo-element */
+  position: relative;
+
+  /* just for case, since size of pseudo-element is 37px */
+  min-width: 37px;
+
+  font-weight: 600;
+  font-size: var(--navigation-font-size);
+  line-height: 1.29;
+}
+.nav_link:after {
+  content: '';
+
+  position: absolute;
+  bottom: -5px;
+  left: 0;
+
+  width: 37px;
+  height: 2px;
+  background-color: var(--green-color);
+
+  /* transition, define it here, and make it visible when :hover */
+  transition: var(--default-trns);
+  transition-property: scale;
+  scale: 0;
+  transform-origin: left;
+}
+.nav_link:hover::after {
+  scale: 1;
+}
+/* #endregion */
+
+/* #region Socials */
+/* Socials, reusable part */
+.main-socials__list {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 12px;
+}
+.main-socials__link {
+  /* apply alpha channel to black color to achieve border at 10% */
+  border: 1px solid rgb(from currentColor r g b / 0.1);
+  border-radius: 10px;
+  padding: 7px; /* note padding here, it's 7 instead of 8, to consider 1px border and preserve total size of item as 44px */
+
```

Comment:
formatted code please (spaces)

Reply:
correct 
.class {
  margin: 0;
  padding: 0;
  height: 10px;
}

.class2 {
  margin: 0;
}

incorrect:
.class {
  margin: 0;
  
  padding: 0;
  height: 10px;
}
.class2 {
  margin: 0;
}

Reply:
will fixed when we will check all project